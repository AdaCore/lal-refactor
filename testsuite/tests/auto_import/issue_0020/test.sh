auto_import -P prj.gpr --source src/main.adb --line 3 --column 10
