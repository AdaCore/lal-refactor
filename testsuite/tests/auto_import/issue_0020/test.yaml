description:
    This test checks that we only propose the specified
    qualifier's unit when there is one..

driver: shell_script
