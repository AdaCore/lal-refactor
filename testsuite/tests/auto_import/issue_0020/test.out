Available Imports:
Import: Ada.Text_IO; Qualifier: 

Edits:
main.adb
1:1-1:1 with Ada.Text_IO;


